src/lib/io/sol-iio.c

```diff
@@ -117,6 +117,8 @@ struct resolve_absolute_path_data {
 #define SAMPLING_FREQUENCY_BUFFER_PATH SYSFS_DEVICES_PATH "/iio:device%d/buffer/sampling_frequency"
 #define SAMPLING_FREQUENCY_TRIGGER_PATH SYSFS_DEVICES_PATH "/trigger%d/sampling_frequency"
 
+#define SYSFS_MOUNT_MATRIX SYSFS_DEVICES_PATH "/iio:device%d/in_mount_matrix"
```

Comment:
From kernel doc, it appears that are other names for this file. Are you going with only `in_mount_matrix` because of accessible devices (for test)?